src/pdf-processor/pdf-pipeline.service.spec.ts

```
@@ -0,0 +1,260 @@
+      callLLM,
+    });
+
+    await expect(
+      pipeline.processPdf(Buffer.from('%PDF'), 'doc.pdf'),
+    ).rejects.toThrow(/incomplete batch/);
+  });
+
+  it('falls back to the section title when metadata path traverses upward', async () => {
+    const callLLM = jest
+      .fn<(...args: unknown[]) => Promise<LlmResponse>>()
+      .mockResolvedValueOnce(llmResponse(`## 안전한 제목\n\n${'본문 '.repeat(700)}`))
```

ESLint

[error] Replace `llmResponse(`##·안전한·제목\n\n${'본문·'.repeat(700)}`)` with `⏎········llmResponse(`##·안전한·제목\n\n${'본문·'.repeat(700)}`),⏎······`

(prettier/prettier)